core/filestorage/filesync/upload.go

```diff
@@ -77,7 +77,8 @@ func (s *fileSync) handleLimitReachedError(err error, it *QueueItem) *errLimitRe
 			log.Error("set limit reached error logged", zap.String("objectId", it.ObjectId), zap.Error(setErr))
 		}
 
-		s.runOnLimitedHook(it.ObjectId, it.FullFileId())
+		bytesLeftPercentage := float64(limitReachedErr.accountLimit-limitReachedErr.totalBytesUsage) / float64(limitReachedErr.accountLimit)
```

Comment:
I'm concerned that accountLimit could be zero due to some network error or bug in limit calculations, maybe add zero value check?